docs/user/supported_devices.rst

```diff
@@ -104,6 +104,7 @@ ar71xx-generic
   - Bullet M2/M5  [#device-class-tiny]_
   - Loco M2/M5  [#device-class-tiny]_
   - Loco M2/M5 XW
+  - Nanostation AC (WA)
```

Comment:
Wrong target. Also, where does the `(WA)` come from?

Reply:
Fixed.

```
root@dc-18e829307296:~# lua -e 'print(require("platform_info").get_image_name())'
ubiquiti-nanostation-ac-wa
```

```
root@dc-18e829307296:~# cat /proc/cpuinfo
system type             : Atheros AR9342 rev 2
machine                 : Ubiquiti Nanostation AC (WA)
processor               : 0
cpu model               : MIPS 74Kc V4.12
BogoMIPS                : 266.64
wait instruction        : yes
microsecond timers      : yes
tlb_entries             : 32
extra interrupt vector  : yes
hardware watchpoint     : yes, count: 4, address/irw mask: [0x0ffc, 0x0ffc, 0x0ffb, 0x0ffb]
isa                     : mips1 mips2 mips32r1 mips32r2
ASEs implemented        : mips16 dsp dsp2
Options implemented     : tlb 4kex 4k_cache prefetch mcheck ejtag llsc dc_aliases perf_cntr_intr_bit nan_legacy nan_2008 perf
shadow register sets    : 1
kscratch registers      : 0
package                 : 0
core                    : 0
VCED exceptions         : not available
VCEI exceptions         : not available
```